tests/actions/test_core_actions.py

```diff
@@ -12,7 +12,7 @@
 import requests
 from requests import Session
 
-from great_expectations.checkpoint.actions import (
+from great_expectations.checkpoint import (
```

Comment:
`ActionContext` is not currently imported to in `great_expectations/checkpoint/__init__.py` so isn't importable like this.